cdn/signed-urls/src/main/java/com/google/cdn/SignedUrls.java

```diff
@@ -75,6 +76,61 @@ public static String getSignature(byte[] privateKey, String input)
   }
   // [END cloudcdn_sign_url]
 
+  // [START cloudcdn_sign_url_prefix]
+  /**
+   * Creates a signed URL with a URL prefix for a Cloud CDN endpoint with the given key. Prefixes
+   * allow access to any URL with the same prefix, and can be useful for granting access broader
+   * content without signing multiple URLs.
+   *
+   * @param requestUrl URL of request
```

Comment:
Applies to all parameters below.
Consider moving these comments to the main method in the new file and refrain from javadoc style comments.

Reply:
Example: https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/dlp/snippets/src/main/java/dlp/snippets/DeIdenitfyWithDeterministicEncryption.java#L42-L58